src/parser/user-extractor-module.ts

```diff
@@ -22,10 +22,10 @@ export class UserExtractorModule implements Module {
   }
 
   /**
-   * Checks if the comment is made by a human user, and not empty.
+   * Checks if the comment is made by a human user, not empty, and not a command.
    */
   _checkEntryValidity(comment: (typeof IssueActivity.prototype.allComments)[0]) {
-    return comment.body && comment.user?.type === "User";
+    return comment.body && comment.user?.type === "User" && comment.body.trim()[0] !== "/";
```

Comment:
I'm surprised that empty comments have been found in testing. I didn't know that is possible.

Reply:
It should not be but according to GitHub types `body` can eventually be `undefined`.